server/cli/src/refresh_dates.rs

```diff
@@ -40,18 +40,6 @@ fn get_timestamp_fields() -> Vec<TableAndFieldName> {
         ("requisition_line", "snapshot_datetime"),
         ("stocktake", "created_datetime"),
         ("stocktake", "finalised_datetime"),
-        ("sync_log", "started_datetime"),
```

Comment:
Removed because it was messing with date refresh cli (which tries to find the latest date, then difference between that date and now and adjusts all dates by that difference, of course sync log had latest date, which was populated with init from reference data cli, thankfully there was a clear message about this, so just moved to ignore sync log datetimes)